Cargo.toml

```diff
@@ -245,6 +245,10 @@ level = 'warn'
 check-cfg = [
     'cfg(mariadb, values(any()))',
     'cfg(sqlite_ipaddr)',
+    'cfg(postgres_12)',
+    'cfg(postgres_13)',
+    'cfg(postgres_14)',
+    'cfg(postgres_15)',
```

Comment:
Confirming what we talked about on Discord:

Let's switch this to match `mariadb`, so `cfg(postgres, values(any()))`

Then this won't need to be updated for future versions.